test/Language/TranslationCompletenessTest.php

```diff
@@ -91,7 +91,13 @@ public function testTranslations()
             }
         }
 
-        // If we have no extra and no missing translations, $err will be empty.
-        self::assertEmpty($err, $err);
+        // Make this test informational only.
+        if ($err !== '') {
+            $this->markTestSkipped("Translation completeness (informational):\n" . $err);
```

Comment:
Is `markTestSkipped()` the right method to use ? Would `markTestIncomplete()` maybe be appropriate ?